Source/Core/UICommon/CommandLineParse.cpp

```diff
@@ -0,0 +1,48 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include <OptionParser.h>
+
+#include "Common/Common.h"
+#include "UICommon/CommandLineParse.h"
+
+namespace CommandLineParse
+{
+std::unique_ptr<optparse::OptionParser> CreateParser(ParserOptions options)
+{
+  auto parser = std::make_unique<optparse::OptionParser>();
+  parser->usage("usage: %prog [options]... [FILE]...").version(scm_rev_str);
+
+  parser->add_option("-v", "--version").action("version").help("Print version and exit");
```

Comment:
This or `video_backend` needs to lose the short switch or get a new one (probably a capital V). Otherwise the command line options become ambiguous.